### PR TITLE
feat(ui-watt): add single selection of datepicker

### PR DESCRIPTION
**Pull request by frontend-specialisten**
## Description

- Rename `watt-date-range-input` -> `watt-datepicker`
- Add support for single date mask / datepicker

## References

- #373

### CODE REVIEW

File: libs/ui-watt/src/lib/components/input/datepicker/+storybook/watt-datepicker-reactive-forms.stories.ts [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+</watt-form-field>
+
+<p>Value: <code>{{exampleFormControlSingle.value | json}}</code></p>
+<p *ngIf="withValidations">Errors: <code>{{exampleFormControlSingle?.errors | json}}</code></p>
+
+<br />
+
+<watt-form-field>
+  <watt-label>Date range</watt-label>
+  <watt-datepicker [formControl]="exampleFormControlRange" range></watt-datepicker>
```
Comment (dzhavat) [inline libs/ui-watt/src/lib/components/input/datepicker/+storybook/watt-datepicker-reactive-forms.stories.ts:61]: issue: in VS Code I get the following error if I simply use the `range` attribute.

image

Let's use the "expanded" version, i.e. `[range]="true"` when displaying a date range picker.

**Review by dzhavat — CHANGES_REQUESTED** [review]

**Review by frontend-specialisten — COMMENTED** [review]
@cla-bot check

**Review by dzhavat — APPROVED** [review]

**Comment by Renetnielsen** [comment]
@cla-bot check

**Comment by dzhavat** [comment]
@cla-bot check

**Comment by frontend-specialisten** [comment]
@cla-bot check

**Comment by frontend-specialisten** [comment]
@cla-bot check

**Pull request merged**